website/assets/css/image-gallery.css

```
@@ -0,0 +1,80 @@
+    margin-right: auto;
+    padding-top: .75em;
+    text-wrap: wrap;
+    font-size: 75%;
+}
+
+.image-gallery gallery-object a figcaption img {
+    padding-top: 0;
+    padding-bottom: 3px;
+}
+
+gallery {
```

Biome (2.4.4)

[error] Unknown type selector is not allowed.

(lint/correctness/noUnknownTypeSelector)